patches/chrome-browser-resources-settings-settings_menu-settings_menu.html.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/chrome/browser/resources/settings/settings_menu/settings_menu.html b/chrome/browser/resources/settings/settings_menu/settings_menu.html
+index 58721ca87c950f2300841cf43f74a9c655e353b3..eafc95fc9e059383a08b7908bcd756d658803710 100644
+--- a/chrome/browser/resources/settings/settings_menu/settings_menu.html
++++ b/chrome/browser/resources/settings/settings_menu/settings_menu.html
+@@ -176,10 +176,12 @@
+             $i18n{multidevicePageTitle}
+           </a>
+ </if>
++<if expr="_google_chrome">
```

Comment:
we're turning off accessibility settings?

Reply:
I'm removing this section because the only entry I could see is adding features by accessing chrome web store.
<img width="694" alt="screen shot 2018-06-18 at 9 18 48 pm" src="https://user-images.githubusercontent.com/4730197/41576624-4922b378-733d-11e8-8614-3575ed3f81bf.png">